sdcm/results_performance.html

```diff
@@ -98,11 +98,15 @@ <h3>Links:</h3>
         <li><a href={{ dashboard_master }}>Kibana</a></li>
         <li><a href="{{ job_url }}">Jenkins job</a></li>
         {% if grafana_snapshot %}
-        <li><a href={{ grafana_snapshot }}>Download Grafana Screenshot</a></li>
+        {% for snapshot in grafana_snapshot %}
+        <li><a href={{ snapshot }}>Download Grafana Screenshot</a></li>
```

Comment:
We need to know which screenshot we are downloading. Please change it to include dashboard name.
To implement this you can, for instance, add the dashboard name to the screenshot filename and then deduce it here or when rendering template
The link, for instance, can have the following name: Download "Per server metrics nemesis" dashboard screenshot*

Reply:
Add the name of dashboard to filename and change the template.